src/main/java/com/entropyteam/entropay/employees/models/Contract.java

```diff
@@ -39,6 +43,9 @@ public class Contract extends BaseEntity {
     @Column
     private Integer vacations;
 
+    @Column
+    private Boolean active;
```

Comment:
I think here we could use the primitive boolean, so that active will never be NULL